_includes/shared/feature-card.html

```diff
@@ -6,8 +6,8 @@
 {% elsif feature.last-updated-release == project.version or feature.last-updated-release == project.version60 %} updated {% endif %}">
     <a class="feature-image-wrapper" href="{{ feature.url }}">
 
-        {% if feature.tags contains 'deprecated' %}<div class="deprecated"></div>
-        {% elsif feature.tags contains 'beta' %}<div class="beta"></div>
+        {% if feature.feature-tags contains 'deprecated' %}<div class="deprecated"></div>
+        {% elsif feature.feature-tags contains 'beta' %}<div class="beta"></div>
```

Comment:
Fixed a bug here where these banners were not working b/c the wrong var name was being referenced.